modules/loader/loader.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import styles from './loader.module.css'
+
+// TODO: If we will need to use loader circle component in different
+// components - move it to Design library
+
```

Comment:
It's ok for now and perfect for this scope. However, we will need a loader for search too... Maybe we can design a more stylish loader in the future and integrate it in design once we work on the Search page